lib/records/I_RecHttp.h

```diff
@@ -201,9 +204,9 @@ class SessionProtocolNameRegistry
   void markIn(const char *value, SessionProtocolSet &sp_set);
 
 protected:
-  unsigned int m_n;         ///< Index of first unused slot.
-  const char *m_names[MAX]; ///< Pointers to registered names.
-  uint8_t m_flags[MAX];     ///< Flags for each name.
+  int m_n = 0; ///< Index of first unused slot.
+  std::array<TextView, MAX> m_names;
+  ts::MemArena m_arena; ///< Storage for non-constant strings.
 
   static uint8_t const F_ALLOCATED = 0x1; ///< Flag for allocated by this instance.
```

Comment:
Is this flag still used?

Reply:
No, that slipped by me and then I wasn't sure it was worth updating the PR. I'll fix that.